lib/appsignal/config.ex

```diff
@@ -104,11 +104,15 @@ defmodule Appsignal.Config do
   """
   @spec valid?() :: boolean
   def valid? do
-    Application.get_env(:appsignal, :config)[:push_api_key]
-    |> empty?
-    |> Kernel.not()
+    do_valid?(Application.get_env(:appsignal, :config)[:push_api_key])
   end
 
+  defp do_valid?(push_api_key) when is_binary(push_api_key) do
+    String.strip(push_api_key) != ""
```

Comment:
I couldn't find `String.strip` in the Elixir docs, because it's deprecated. The CI prompts this warning (weird that our `mix compile --warnings-as-errors` job doesn't fail on this):

```
warning: String.strip/1 is deprecated. Use String.trim/1 instead
  lib/appsignal/config.ex:111: Appsignal.Config.do_valid?/1
```

I can find the `Strip.trim` function way back in the 1.3.0 docs, so I think all our supported Elixir version will have the `String.trim` function. No need to use one function on older versions off Elixir.
https://hexdocs.pm/elixir/1.3.0/String.html#trim/1

Looks like it was removed in 1.3.0, because the last reference I can find to `String.strip` is in 1.2.6: https://hexdocs.pm/elixir/1.2.6/String.html#strip/1